webhook_receiver_shopify/views.py

```diff
@@ -66,6 +67,20 @@ def inner(request):
     return inner
 
 
+def validate_webhook_payload(payload):
```

Comment:
Can you change this method name to `validate_order_tags` since it's not actually validating the entire payload?

Reply:
Updated. Thanks, Pooja!